dts/msm8939-asus-z00t.dts

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "msm8916.dtsi"
+
+/ {
+	compatible = "qcom,msm8939-mtp", "qcom,msm8939";
+	qcom,msm-id = <239 0 241 0 263 0>;
```

Comment:
you should just pick one of them, probably `qcom,msm-id = <239 0>;` for MSM8939

Reply:
I meant what I said, this still looks like before :D

Reply:
@konradybcio FYI, I still think you should change this. Your device doesn't use APQ8039.